src/main/java/com/hcl/appscan/jenkins/plugin/scanners/StaticAnalyzer.java

```diff
@@ -201,7 +201,10 @@ public void validateSettings(JenkinsAuthenticationProvider authProvider, Map<Str
             if (properties.containsKey(CoreConstants.INCLUDE_SCA) && properties.containsKey(CoreConstants.UPLOAD_DIRECT) && !properties.get(TARGET).endsWith(".irx")) {
                 throw new AbortException(Messages.error_invalid_format_include_sca());
             }
-
+            
+            if(properties.containsKey(CoreConstants.SCAN_ID) && properties.containsKey(CoreConstants.INCLUDE_SCA)){
+                 properties.remove(CoreConstants.INCLUDE_SCA);
```

Comment:
The check `properties.containsKey(CoreConstants.INCLUDE_SCA` in unnecessary. The call to remove() will be a no-op if the key doesn't exist.

Reply:
Okay, Matt.